src/main/java/eu/dissco/backend/configuration/ApplicationConfiguration.java

```diff
@@ -12,6 +13,7 @@ public class ApplicationConfiguration {
   @Bean
   public ObjectMapper objectMapper() {
     var mapper = new ObjectMapper().findAndRegisterModules();
+    mapper.setDateFormat(new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS"));
     mapper.setSerializationInclusion(Include.NON_NULL);
```

Comment:
In the handle server, I have this as 'yyyy-MM-dd'T'HH:mm:ss.SSS'Z'' (with the Z at the end) to indicate UTC time. That's the pattern from our discussion earlier.

Reply:
Ah I don't think that is correct. You hardcode the Z then at the end which doesn't always have to be true. I think then it should be `yyyy-MM-dd'T'HH:mm:ss.SSSXXX`. However it still gives me five behind the second so for some reason it is not working as expected and I have to dig into this some more to get the formatting right

Reply:
Okay, I'll align the handle server with this for now, let me know if anything changes